packages/p2p-v2/src/pages/buy-sell/screens/BuySell/BuySell.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { BuySellTable } from '../BuySellTable';
+
+const BuySell = () => {
+    return (
+        <div>
+            <BuySellTable />
+        </div>
```

Comment:
Is the `<div></div>` needed @ameerul-deriv?
```suggestion
      <BuySellTable />
```

Reply:
for now I left it cause I'll be adding more here when other features need to be added to the BuySell page